second-gen/packages/core/components/badge/Badge.base.ts

```diff
@@ -55,8 +58,6 @@ export type FixedValues = (typeof FIXED_VALUES)[number];
 /**
  * @element sp-badge-base
  * @property {BadgeVariant} variant - The variant of the badge.
- * @property {boolean} subtle - Whether the badge is subtle.
```

Comment:
This only exists in S2's new styles so I'm testing out moving these into the S2-only component.